app/services/et_api_handler.rb

```diff
@@ -18,6 +18,17 @@ def self.submit(form_hash)
     { data: http_response.parsed_response, status: http_response.code }
   end
 
+  def self.validate_response(attribute, value)
+    response = HTTParty.
+               post("#{ENV.fetch('ET_API_URL', 'http://api.et.127.0.0.1.nip.io:3100/api')}/v2/validate_response",
+                 body: {
+                   "#{attribute}": value
```

Comment:
What you are doing here - is converting 'attribute' from whatever it is - into a string - then into a symbol to be used as as key.  Are you meaning to do this for some reason ?